Themes/default/languages/PersonalMessage.english.php

```diff
@@ -36,6 +36,7 @@
 $txt['reply_to_all'] = 'Reply to All';
 $txt['delete_conversation'] = 'Delete Conversation';
 $txt['remove_conversation'] = 'Remove all messages in this conversation?';
+$txt['personal_message'] = 'Personal Message';
```

Comment:
String already declared in ManageMail.english.php L27

Reply:
The two language files are never loaded at the same time, so it shouldn't make a difference. But to avoid even a hypothetical conflict, I suppose I could change the key of this one from 'personal_message' to just 'pm'.

Reply:
perhaps "export_personal_message" that way we can know some context on where its being used.

Reply:
Good idea, @MissAllSunday.

Reply:
Hm. Actually, I'd changed this to just `$txt['pm']` at one point. It seems that I accidentally reverted that during the major rewrite in 21b8918. I'll restore that now.